core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Run.java

```diff
@@ -559,4 +548,12 @@ public boolean userCanWrite(User user) {
     return changeLogs;
   }
 
+  public Boolean getPairedEnd() {
+    return null;
+  }
+
+  public void setPairedEnd(boolean pairedEnd) {
+    // do nothing;
```

Comment:
Getter is fine for convenience, but either remove the setter or throw `UnsupportedOperationException` instead of ignoring misuse that can cause confusion